examples/dns-views-compatibility.py

```diff
@@ -0,0 +1,150 @@
+"""
+DNS views allow for <blurb here>
+
+This means an FQDN can now appear in more than one zone. We have implemented
+views in a back-compatible way, so if you do not intend to use them, you can
+largely continue to use the API as you've done before.
+
+COMPATIBILITY
+
+Until now, the unique identifier of a zone in the API has been the FQDN. This
+appears both in the URL for API requests and in the 'zone' field of zones and
+records. The API throws an error if the URL and field in the body do not match.
+
+Going forward, the unique identifier of the zone is "decoupled" from the 'zone'
+field. The reference in the URL can now be an arbitratry string, unique within
+your account. This identifier is passed and returned in the 'name' field for
+zones, and the 'zone_name' field for records. You may of course continue to use
+the FQDN as the identifier - in fact, if you are not using "views", it is
+recommended that you do so. However:
+
+* Mismatches between the zone name in the URL and the 'zone' field are no
+  longer validated to match, as it is no longer an invalid condition.
+* New field 'name' on zone responses
+* New field 'zone_name' on record responses
+
+Crucially, if you are NOT using "views", then the 'zone' and 'name' fields on
+the zone will probably have the same value, but the unique identifier for the
+zone is ALWAYS the value in the 'name' field. So in general, code should be
+written or updated to prefer the 'name/zone_name' fields for identification,
+and when using "views" functionality, calling code MUST be
+"name/zone_name aware".
```

Comment:
yep, agreed. i started making noise about this a while ago and I believe things are in motion. this stuff may help a bit for building that, and as i soon as i can point somewhere else instead of writing things here, we will be :)